compute_endpoint/globus_compute_endpoint/engines/high_throughput/engine.py

```diff
@@ -19,6 +19,9 @@
 
 import dill
 from globus_compute_common import messagepack
+from globus_compute_common.messagepack.message_types import (
+    EPStatusReport as CommonEPStatusReport,
+)
```

Comment:
I'm not looking at the entire file at the moment; is there a need to alias this differently?  Or otherwise "add this hunk and associated change to this PR"?

Reply:
These points should cover the reasoning.
- Both `globus_compute_endpoint` and `globus_compute_common` have an `EPStatusReport` class.
- The `HighThroughputEngine` class inherits from `GlobusComputeEngine`.
- `GlobusComputeEngine.get_status_report()` has the `EPStatusReport` from `globus_compute_common` as its return type.
- `HighThroughputEngine.get_status_report()` must use the same return type to keep mypy happy.
- `HighThroughputEngine` also utilizes the `EPStatusReport` from `globus_compute_endpoint` elsewhere in the same module.

Reply:
Heh.  Ugh.  :+1: